### DISCUSSION TITLE
Mistral

### DISCUSSION THREAD

**Discussion by Boganabi** [DrAlzahraniProjects/csusb_fall2024_cse6550_team1#18] [General]
Discuss things related to Mistral here!

**Comment by itsnotmik** [comment]
Mistral AI documentation and guides available here: https://docs.mistral.ai

**Comment by ACraig7** [comment]
Mistral is [open source](https://huggingface.co/docs/transformers/en/model_doc/mistral#license).

**Comment by ACraig7** [comment]
@itsnotmik On this [page](https://docs.mistral.ai/deployment/laplateforme/tier/#:~:text=To%20prevent%20misuse%20and%20manage%20the%20capacity%20of,limits%3A%20Requests%20per%20second%20%28RPS%29%20Tokens%20per%20minute%2Fmonth) at the bottom, there is this [link](https://console.mistral.ai/limits/) that can show your rate limits based on the workspace you created. It is under the **Usage tiers** section. It will interesting to see what the rate limit is cause it may be high enough that we do not have to worry about it.
